frontend/src/views/ManagementPage/UserTools.vue

```diff
@@ -99,7 +99,8 @@
               target="_blank"
               outlined
               color="primary"
-              rel="noopener"
+              rel="noopener external"
+              title="Decouvrir nos ressources - ouvre une nouvelle fenêtre"
```

Comment:
Petite coquille (en bas aussi)... _d**é**couvrir_

```suggestion
              title="Découvrir nos ressources - ouvre une nouvelle fenêtre"
```